locust/runners.py

```diff
@@ -351,7 +352,9 @@ def start_shape(self) -> None:
     def shape_worker(self) -> None:
         logger.info("Shape worker starting")
         while self.state == STATE_INIT or self.state == STATE_SPAWNING or self.state == STATE_RUNNING:
-            new_state = self.environment.shape_class.tick() if self.environment.shape_class is not None else None
+            new_state: Union[Tuple[int, float], Tuple[int, float, Optional[List[Type[User]]]], None] = (
```

Comment:
This type info hurts my head :) (I get it, it just hurts...)

Probably the variable should be renamed (because it is confusing to have self.state on the line above and then "new_state"). Maybe next_tick? or new_tick? 

Probably we should have defined a proper type that tick returns instead of just a tuple, but that is not your fault :)

Reply:
Okay, should we still rename `new_state` to next tick?

Reply:
or maybe just `current_tick`?

Reply:
👍